fix(aws): document nRF Cloud team ID (#391)
NordicSemiconductor/asset-tracker-cloud-docs: docs/aws/nRFCloudLocationServices.rst

nRF Cloud Location Services
###########################

Prerequisites
*************

> Any nRF Cloud Location Service requires the configuration of your team ID.

Reviewer: ```suggestion
To use any nRF Cloud Location Service, you need to configure your team ID.
```

> Use the following command to configure your team ID:

Reviewer: ```suggestion
Run the following command to configure your team ID:
```
